assets/src/components/detours/detourMap.tsx

```diff
@@ -196,6 +198,15 @@ export const DetourMap = ({
                 interactionStatesDisabled={false}
               />
             ))}
+            {missedStops.map((stop) => (
+              <StopMarkerWithStopCard
+                key={stop.name}
+                stop={stop}
+                zoomLevel={zoomLevel}
+                interactionStatesDisabled={false}
+                missed
+              />
+            ))}
```

Comment:
note(non-blocking): Just pointing this out so we're aware of it, but I imagine we'll get to this at a later point in this epic. If my understanding is correct, the tab order will be somewhat weird because the stops are rendered in two separate chunks, so it'll first tab over all the non-missed stops, and then it'd tab over the missed stops, and I'm not sure that's what we want (but I don't actually know what we want)

We'd probably need to re-merge these stop lists based on stop sequence and render them in order.

Reply:
Ah yeah that's a good point. 

Do you think it would make sense for `useDetour` to return a collection of things that are like `Stop[]`, but also have a `missed` attribute?

Reply:
Thinking about it more, this is a good idea, and I think there's an easy way to accomplish this. Stay tuned!

Reply:
@firestack better approach achieved, and that meant we can get rid of https://github.com/mbta/skate/pull/2482!